docs/shared/continue-as-new.md

```diff
@@ -1,6 +1,6 @@
 Temporal stores the execution history of all Workflows.
 There is a maximum limit of this execution history (50,000 events).
-Even though Temporal Server emits warnings while your workflow are approaching this limit (every 10,000 events), you should make sure
+Even though Temporal Server emits warnings while your Workflows are approaching this limit (every 10,000 events), you should make sure
```

Comment:
FYI - deleting this file here https://github.com/temporalio/documentation/pull/832